index.js

```diff
@@ -86,6 +87,9 @@ function addAliases (aliases) {
 
 function addAlias (alias, target) {
   moduleAliases[alias] = target
+  // Cost of sorting is lower here than during resolution
+  moduleAliasNames = Object.keys(moduleAliases)
```

Comment:
Ideally I had wanted to avoid keeping a separate list of alias names, but alas it's not sensible. The alternative would be evaluating the keys of the object and sorting them every time a resolution is made, which incurs a far larger performance penalty. Keeping the `Object.prototype.keys()` evaluation and subsequent sort here means that it runs a handful of times, once for each alias added. Performing it during resolution would mean evaluating it once for every single file require'd which would take far more time overall.